src/main/scala/org/dbpedia/spotlight/wikistats/JsonPediaParser.scala

```diff
@@ -135,16 +135,21 @@ class JsonPediaParser(inputWikiDump: String, lang: String)
     Input:  - None
     Output: - RDD of all wikiId and the article text
    */
-  def getArticleText(): RDD[(Long, String)] = {
 
-    dfWikiRDD.select("wid","wikiText","type")
+  def getArticleText(): RDD[(Long, String, List[(String, Long, String)])] = {
```

Comment:
Somehow this case class was always giving some casting errors. So decided just declare variable.